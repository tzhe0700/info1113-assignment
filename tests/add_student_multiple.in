ADD STUDENT Hermione Granger
ADD STUDENT Ron Weasley
STUDENT 100000
STUDENT 100001
EXIT

