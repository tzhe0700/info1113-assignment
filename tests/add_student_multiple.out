Success.
Success.
100000: Hermione Granger
100001: Ron Weasley

